source/message11.rst

```diff
@@ -23,9 +23,9 @@ This partly annotated version of the image was made using :ref:`code from messag
 Interpretation
 --------------
 
-.. todo::
+Operator 448 is consistent with checking whether the first number equal than the second number. We will denote it as `eq`.
 
-   Add an interpretation for the eleventh message.
+Operator 2 is consistent with `true` and operator 8 is consistent with `false`.
```

Comment:
Would you mind changing `true` and `false` to `t` and `f`, respectively? Notice this is what's already used in the decoded message, and it's currently what's generated by the `annotate.hs` program.

Reply:
You're right